otterdog/adoptium.jsonnet

```diff
@@ -214,6 +256,11 @@ orgs.newOrg('adoptium') {
           secret: "pass:bots/adoptium/github.com/ci-webhook-secret",
         },
       ],
+      secrets: [
+        orgs.newRepoSecret('JENKINS_WORKFLOW_SECRET') {
```

Comment:
not sure what the value of this is, ping @smlambert can someone provide @netomi with the actual value?